code/parachain/frame/cosmwasm/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::too_many_arguments)]
+#![allow(clippy::unnecessary_mut_passed)]
+
+use codec::Codec;
+use sp_std::collections::btree_map::BTreeMap;
+#[cfg(not(feature = "std"))]
+use sp_std::vec::Vec;
+
+// Cosmwasm Runtime API declaration. Implemented for each runtime at
```

Comment:
afaik picasso and composable have no cosmwasm so cannot be implemented. or these are todo!()?